src/js/components/Toolbar/Toolbar.tsx

```diff
@@ -57,26 +62,34 @@ const Toolbar: React.FunctionComponent<
 
   return (
     <div className="Grid Toolbar">
-      <div className="Grid-cell u-size3of4">
-        {showSearch && (
-          <Field
-            formFieldProps={{
-              placeholder: "Find a program, season, or member",
-              leftIcon: <Icon className="Icon" name="search" />,
-            }}
-            name="Sample"
-          />
-        )}
-      </div>
-      <div className="Grid-cell u-size1of4 u-flex u-flexAlignContentCenter u-flexJustifyEnd u-padXs flyout-container">
+      {children && (
+        <div className="Grid-cell u-flexAlignSelfCenter u-sizeFit u-textNoWrap u-textEllipsis u-spaceRightLg">
+          {children}
+        </div>
+      )}
+
+      {showSearch && (
+        <div className="Grid-cell u-flexAuto">
+            <Field
+              type="input"
+              formFieldProps={{
+                placeholder: "Find a program, season, or member",
+                leftIcon: <Icon className="Icon" name="search" />,
+              }}
+              name="Sample"
+            />
+        </div>
+      )}
```

Comment:
We should probably remove this. The correct way to add a search after these changes would be to use component composition just like you did with the breadcrumbs example. Do you mind removing this piece and any associated bits (and bytes)?

Reply:
@bradcypert, I've removed the search component from the Toolbars component and included it as a part of the Toolbar stories. Thanks.